--- iptcutil.c	2020-06-16 18:47:35.486067200 +0200
+++ iptcutil_new.c	2020-06-16 18:52:53.149818500 +0200
@@ -236,10 +236,10 @@
 
       /* we found the 0x1c tag and now grab the dataset and record number tags */
       dataset = getc(ifile);
-      if ((char) dataset == EOF)
+      if (dataset == EOF)
         return -1;
       recnum = getc(ifile);
-      if ((char) recnum == EOF)
+      if (recnum == EOF)
         return -1;
       /* try to match this record to one of the ones in our named table */
       for (i=0; i< tagcount; i++)
